packages/zowe-explorer/src/command/UnixCommandHandler.ts

```diff
@@ -0,0 +1,269 @@
+            cmdArgs[prop] = profile[prop];
+        }
+        return cmdArgs;
+    }
+
+    public async issueUnixCommand(session?: imperative.Session, command?: string, node?: IZoweTreeNode): Promise<void> {
+        let cwd: string;
+        let profile: imperative.IProfileLoaded;
+        if (node) {
+            cwd = node.fullPath;
+            await this.checkCurrentProfile(node);
+            if (!session) {
+                session = ZoweExplorerApiRegister.getUssApi(node.getProfile()).getSession();
+                if (!session) {
+                    return;
+                }
+            }
+        }
+        if (!session) {
+            const profiles = Profiles.getInstance();
+            const allProfiles: imperative.IProfileLoaded[] = profiles.allProfiles;
+            const profileNamesList = allProfiles.map((temprofile) => {
+                return temprofile.name;
+            });
+            if (profileNamesList.length) {
+                const quickPickOptions: vscode.QuickPickOptions = {
+                    placeHolder: localize("issueUnixCommand.quickPickOption", "Select the Profile to use to submit the Unix command"),
+                    ignoreFocusOut: true,
+                    canPickMany: false,
+                };
+                const sesName = await Gui.showQuickPick(profileNamesList, quickPickOptions);
+                if (sesName === undefined) {
+                    Gui.showMessage(localize("issueUnixCommand.cancelled", "Operation Cancelled"));
+                    return;
+                }
+                profile = allProfiles.filter((temprofile) => temprofile.name === sesName)[0];
+                if (cwd == undefined) {
+                    cwd = await vscode.window.showInputBox({
+                        prompt: "Enter the path of the directory in order to execute the command",
+                        value: "",
+                    });
+                }
+                if (!node) {
+                    await Profiles.getInstance().checkCurrentProfile(profile);
+                }
+                if (Profiles.getInstance().validProfile !== ValidProfileEnum.INVALID) {
+                    session = ZoweExplorerApiRegister.getUssApi(profile).getSession();
+                } else {
+                    Gui.errorMessage(localize("issueUnixCommand.checkProfile", "Profile is invalid"));
+                    return;
+                }
+            } else {
+                Gui.showMessage(localize("issueUnixCommand.noProfilesLoaded", "No profiles available"));
+                return;
+            }
+        } else {
+            profile = node.getProfile();
+        }
+        if (ZoweExplorerApiRegister.getCommandApi(profile).sshProfileRequired) {
+            this.sshSession = await this.setsshSession();
+        } else {
+            Gui.showMessage(localize("issueUnixCommand.notsupportedForProfile", "Action not being supported for the profile type ") + profile.type);
```

Comment:
lets make this string consistent with the others where the api doesn't exist for the extender's profile type, ie.
Gui.errorMessage(localize("issueMvsCommand.apiNonExisting", "Not implemented yet for profile of type: ") + profile.type); 
from MVS command.

Reply:
Added {path} to the output view. And wrapping the path in quotes helped to resolve the error. Modified the above Gui.errorMessage